frontend/public/kubevirt/components/nic.jsx

```diff
@@ -85,8 +95,9 @@ const NIC_TYPE_VM = 'nic-type-vm';
 const NIC_TYPE_CREATE = 'nic-type-create';
 
 export const NicRow = (onChange, onAccept, onCancel) => ({obj: nic}) => {
+  const namespace = getNamespace(nic.vm || nic.vmTemplate);
```

Comment:
@suomiy @rawagner hi, I made a mistake here :-(

`nic.vm` can be defined ( actually it is a required prop, so it is always defined ) but be without namespace label, in this case:

`getNamespace(nic.vm || nic.vmTemplate)` will be empty while `getNamespace(nic.vm) || getNamespace(nic.vmTemplate)` will hold the namespace of the vm template.

I will have to fix that in a new PR :-(

Do you know if we can make `vm` a none required prop, so if called from a template page it can be `undefined` ?
Should I just use the longer phrase ?
Help :-)

Reply:
or something like this 

```js
getNamespace(nic.vmTemplate || nic.vm) // order important (vm might be template vm)
```

or the original approach but I would add some comment so nobody refactors that ;)

Please also add the comment to the props that vm can be template vm (was not yet created)

Reply:
:+1: Thanks, the '(nic.vmTemplate || nic.vm)' looks nice to me, I currently do not have a running okd :-( , will make a PR ASAP.